cirq-google/cirq_google/study/device_parameter.py

```diff
@@ -46,16 +46,19 @@ class DeviceParameter(SupportsDeviceParameter):
            entry in a list.
        idx: If this key is an array, which index to modify.
        value: value of the parameter to be set, if any.
+       units: string value of the unit type of the value, if any.
+          For instance, "GHz", "MHz", "ns", etc.
     """
 
     path: Sequence[str]
     idx: Optional[int] = None
     value: Optional[Any] = None
+    units: Optional[str] = None
```

Comment:
Is there a plan for exposing the spec for units? i.e. how do users know the valid options here? If there's a limited subset of Pyle units that we support, might it be worth providing those explicitly?

Reply:
To answer this comment a bit belatedly, there isn't an immediate plan to expose the units.  However, there is also no plan to expose the valid path either.  Users will generally need an expert internal user in order to use this feature anyway.

The valid spec is to use labrad units, which we may eventually expose, but this is a longer term project.  We probably should revisit this when we start looking at how we want to integrate services like reg/DV with cirq_google.